front/src/reducers/osrdconf/osrdConfCommon/__tests__/utils.ts

```diff
@@ -600,6 +602,224 @@ const testCommonConfReducers = (slice: OperationalStudiesConfSlice | StdcmConfSl
+      coordinates: [47.99542250806296, 0.1918181738752042],
+      location: {
+        track_section: '60ca8dda-6667-11e3-81ff-01f464e0362d',
+        offset: 426.443,
+      },
+    });
+    const parisVia = testDataBuilder.buildPointOnMap({
+      id: 'paris',
+      coordinates: [48.904852473668086, 2.4369545094357736],
+      location: {
+        track_section: '63c905ee-6667-11e3-81ff-01f464e0362d',
+        offset: 719.258,
+      },
+    });
+
+    it('should handle insertion for a route with no existing via and no origin', () => {
+      const pathSteps = [null, strasbourg];
+      const store = createStore(slice, {
+        pathSteps,
+      });
+
+      store.dispatch(slice.actions.addViaV2(parisVia));
+      const state = store.getState()[slice.name];
+      expect(state.pathSteps).toStrictEqual([null, paris, strasbourg]);
+    });
+
+    it('should handle insertion for a route with no existing via and no destination', () => {
+      const pathSteps = [brest, null];
+      const store = createStore(slice, {
+        pathSteps,
+      });
+
+      store.dispatch(slice.actions.addViaV2(rennesVia));
+      const state = store.getState()[slice.name];
+      expect(state.pathSteps).toStrictEqual([brest, rennes, null]);
+    });
+
+    it('should handle insertion for a route with no existing via', () => {
+      const pathSteps = [brest, strasbourg];
+      const store = createStore(slice, {
+        pathSteps,
+      });
+
+      store.dispatch(slice.actions.addViaV2(lemansVia));
+      const state = store.getState()[slice.name];
+      expect(state.pathSteps).toStrictEqual([brest, lemans, strasbourg]);
+    });
+
+    it('should correctly append a new via point when the existing via is closer to the origin', () => {
+      const pathSteps = [brest, rennes, strasbourg];
+      const store = createStore(slice, {
+        pathSteps,
+      });
+
+      store.dispatch(slice.actions.addViaV2(lemansVia));
+      const state = store.getState()[slice.name];
+      expect(state.pathSteps).toStrictEqual([brest, rennes, lemans, strasbourg]);
+    });
+
+    it('should insert a via between two existing ones based on distance from origin', () => {
+      const pathSteps = [brest, rennes, paris, strasbourg];
+      const store = createStore(slice, {
+        pathSteps,
+      });
+
+      store.dispatch(slice.actions.addViaV2(lemansVia));
+      const state = store.getState()[slice.name];
+      expect(state.pathSteps).toStrictEqual([brest, rennes, lemans, paris, strasbourg]);
+    });
+
+    it('should insert a via at the end of the route', () => {
+      const pathSteps = [brest, rennes, lemans, strasbourg];
+      const store = createStore(slice, {
+        pathSteps,
+      });
+
+      store.dispatch(slice.actions.addViaV2(parisVia));
+      const state = store.getState()[slice.name];
+      expect(state.pathSteps).toStrictEqual([brest, rennes, lemans, paris, strasbourg]);
+    });
+  });
+
+  it('should handle permuteViasV2', () => {
```

Comment:
Hum reading the test made me realize that "permute" may not be the right word, since we do not permute 2 vias, but move 1 right ?
```suggestion
  it('should handle permuteViasV2', () => {
```

Reply:
Yeah good idea :) Done.